gitbootcamp.rst

```diff
@@ -551,3 +551,35 @@ To edit an open pull request that targets ``main``:
       git push git@github.com:<contributor>/cpython <pr_number>:<branch_name>
 
 5. Optionally, :ref:`delete the PR branch <deleting_branches>`.
+
+
+GitHub CLI
+----------
+
+GitHub CLI is a CLI (command-line interface) to work with GitHub.
+For example, you can create pull requests and releases,
+set alias commands, check PR statuses.
```

Comment:
This could be simplified further:
```suggestion
`GitHub CLI <https://cli.github.com/>`_ is a command-line
interface that allows you to create, update, and check GitHub
issues and pull requests.
```
This also adds a link and removes releases, since they are not relevant for our workflow.

Reply:
Addressed